nautobot_golden_config/management/commands/run_generate.py

```diff
@@ -0,0 +1,22 @@
+"""Add the run_generate command to nautobot-server."""
+
+from django.core.management.base import BaseCommand
+from nautobot.extras.jobs import get_job
+
+from nautobot_golden_config.management.commands.utils import job_runner
+
+
+class Command(BaseCommand):
+    """Boilerplate Command to inherit from BaseCommand."""
+
+    help = "Run Intended Job"
```

Comment:
Help and docstring are a bit vague, I think.

Reply:
Help I agree with, but docstring, making each unique and still letting pydocstyle pass is a bit cumbersome, but willing to take suggestions.